Makefile

```diff
@@ -34,6 +34,7 @@ stop:
 start-ecr:
 	@echo "-- Start all components --"
 	docker-compose -f ecr.docker-compose.yml up -d
+	timeout 1m ./grafana_healthcheck.sh || echo Grafana startup timed out
```

Comment:
Nice 👍 Didn't known `timeout` command existed!

Reply:
Yeah it's part of GNU coreutils - doesn't exist on Mac but you can install it with home-brew